packages/web-react/src/hooks/useDeprecatedMessage.ts

```diff
@@ -1,35 +1,5 @@
 import { useEffect } from 'react';
 
-/**
```

Comment:
@tomassychra I removed this hook as you mentioned. I didn't feel competent to improve the decision tree in the useDeprecationMessage hook, so I will leave it up to you 💪.